spec/requests/authors_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Authors", type: :request do
+  let(:author) { create(:author) }
```

Comment:
By adding `!` it creates the author automatically in the database so that you don't need to call the author variables explicitly in the before.